testing/test_conftest.py

```diff
@@ -276,73 +279,73 @@ def fixture():
     os.path.normcase("x") != os.path.normcase("X"),
     reason="only relevant for case insensitive file systems",
 )
-def test_conftest_badcase(testdir):
+def test_conftest_badcase(pytester: Pytester) -> None:
     """Check conftest.py loading when directory casing is wrong (#5792)."""
-    testdir.tmpdir.mkdir("JenkinsRoot").mkdir("test")
+    pytester.mkdir("JenkinsRoot").mkdir("test")
     source = {"setup.py": "", "test/__init__.py": "", "test/conftest.py": ""}
-    testdir.makepyfile(**{"JenkinsRoot/%s" % k: v for k, v in source.items()})
+    pytester.makepyfile(**{"JenkinsRoot/%s" % k: v for k, v in source.items()})
 
-    testdir.tmpdir.join("jenkinsroot/test").chdir()
-    result = testdir.runpytest()
+    pytester.path.joinpath("jenkinsroot/test").chdir()
```

Comment:
> testing/test_conftest.py:286: error: "Path" has no attribute "chdir" [attr-defined]

```Path``` does not have attribute ```chdir```. You can acheive same by using ```os.chdir(path)```.
```suggestion
    os.chdir(pytester.path.joinpath("jenkinsroot/test"))
```